documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -209,6 +209,8 @@ You migrate virtual machines (VMs) from the command line (CLI) by creating {proj
 You must specify a name for cluster-scoped CRs.
 
 You must specify both a name and a namespace for namespace-scoped CRs.
+
+You must have an {virt} service account token with `cluster-admin` privileges.
```

Comment:
right but the way it's phrased is too restrictive - the token is only needed for adding a **remote** cluster as a provider
to be more concrete, if we migrate from vSphere to the cluster that MTV is deployed on, then we don't have to have a service account since we can add the 'local' cluster without specifying a service account (and we don't need one for the vSphere provider because its credentials are different)
```suggestion
In order to migrate to or from an {openshift} cluster that is different than the one the migration plan is defined on, you must have an {virt} service account token with `cluster-admin` privileges.
```

Reply:
thinking out loud: I was not sure about "different than the one the migration is define on" part, but I don't want to say here "different than the one the MTV operator is deployed on" because the MTV operator may be deployed also on the remove cluster. maybe we should refer to the 'local' cluster as the cluster than the migration is triggered from?...

Reply:
I think your suggestion, "In order to..you must" is clear enough, but open to other ideas. I don't have anything.